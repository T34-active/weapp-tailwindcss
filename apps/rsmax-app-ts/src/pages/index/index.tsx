import { Image, Text, View } from 'rsmax/one'
import styles from './index.css'

export default () => {
  return (
    <View className={styles.app}>
      <View className={styles.header}>
        <Image
          src="https://gw.alipayobjects.com/mdn/rms_b5fcc5/afts/img/A*OGyZSI087zkAAAAAAAAAAABkARQnAQ"
          className={styles.logo}
        />
        <View className={styles.text}>
          编辑
          {' '}
          <Text>src/pages/index/index.js</Text>
          开始
        </View>
      </View>
    </View>
  )
}

// TypeError: Cannot read property 'current' of undefined
//     at li.callLifecycle (createPageConfig.js:142)
//     at li.onLoad (createPageConfig.js:100)
//     at l.<anonymous> (VM496 WASubContext.js:1)
//     at VM496 WASubContext.js:1
//     at X (VM496 WASubContext.js:1)
//     at VM496 WASubContext.js:1
//     at Array.forEach (<anonymous>)
//     at VM496 WASubContext.js:1
//     at Array.forEach (<anonymous>)
//     at VM496 WASubContext.js:1(env: macOS,mp,1.06.2503300; lib: 3.8.7)
