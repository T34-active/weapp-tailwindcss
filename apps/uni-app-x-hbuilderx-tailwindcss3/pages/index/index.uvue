<template>
	<view class="px-4">
		<view class="mt-[32.43rpx] bg-[#322323]">
			<text class="text-[#844343]">{{title}}</text>
		</view>
	</view>
</template>

<script>
	export default {
		data() {
			return {
				title: 'Hello'
			}
		},
		onLoad() {

		},
		methods: {

		}
	}
</script>

<style>

</style>