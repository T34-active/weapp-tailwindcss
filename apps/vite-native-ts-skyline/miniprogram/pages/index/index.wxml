<!-- index.wxml -->
<navigation-bar title="Weixin" back="{{false}}" color="black" background="#FFF"></navigation-bar>
<view class="min-h-screen bg-slate-950">
  <view class="p-8">
    <view class="i-mdi-home bg-[#43ffff] text-[200px]"></view>
    <view class="usermotto">
      <text class="user-motto text-[#234fff] bg-[#43ffff]">{{motto}}</text>
    </view>
  </view>
</view>