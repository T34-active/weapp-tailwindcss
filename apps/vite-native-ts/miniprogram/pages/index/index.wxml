<view class="p-4">
  <IceButton class="bg-[#d72929]" tw-class="bg-[orange]" hover-class="bg-[green]" tw-hover-class="bg-[pink]"></IceButton>
</view>