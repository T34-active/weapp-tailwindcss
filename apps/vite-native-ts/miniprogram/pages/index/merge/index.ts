import { twMerge } from '@weapp-tailwindcss/merge/v3'
import { twMerge as twMergeV4 } from '@weapp-tailwindcss/merge/v4'

Page({
  data: {
    mergedClass: twMerge('p-1 p-2 p-0.5 text-[34px] text-[#ececec]'),
    mergedClassV4: twMergeV4('p-1 p-2 p-0.5 text-[34px] text-[#ececec]'),
  },
})
