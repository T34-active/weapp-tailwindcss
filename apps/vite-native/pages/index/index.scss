// https://sass-lang.com/documentation/at-rules/import/#plain-css-imports
/* stylelint-disable-next-line import-notation */
// @import url('../../app.scss');

// @weapp-vite-keep-import 'xx';

// @use '../../app.wxss';

.s {
  .a {
    color: turquoise;
  }
}

.user-motto {
  font-size: 12px;
}
