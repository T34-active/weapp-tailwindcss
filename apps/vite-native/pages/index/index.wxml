<scroll-view scroll-y type="list">
  <view class="space-y-2.5">
    <view class="bg-[#7d7ac2] text-[100px] text-[#123456] w-[323px] {{true?'h-[30px]':'h-[45px]'}}">
      111
    </view>
    <view class="bg-[#3a32d1] text-[100rpx] text-red-700 w-[323px] {{false?'h-[30px]':'h-[45px]'}}">
      222
    </view>
    <view class="bg-gradient-to-b from-[#2f73f1] to-[#4bcefd] w-[323px] h-[30px]"></view>
    <view class="bg-gradient-to-t from-[#2f73f1] to-[#4bcefd] w-[323px] h-[30px]"></view>
    <view class="bg-gradient-to-tr from-[#2f73f1] to-[#4bcefd] w-[323px] h-[30px]"></view>
    <view class="space-x-2.5">
      <view class="inline-block h-10 w-[50px] bg-amber-300 border-4"></view>
      <view class="inline-block h-10 w-[50px] bg-amber-300"></view>
      <view class="inline-block h-10 w-[50px] bg-amber-300"></view>
      <view class="inline-block h-10 w-[50px] bg-amber-300"></view>
    </view>
    <view class="i-mdi-home"></view>
    <view class="hover:border-blue-200/20">111</view>
    <view class='size-12 bg-blue-500/30 text-pink-300'>BLUE BLOCK</view>
  </view>
</scroll-view>