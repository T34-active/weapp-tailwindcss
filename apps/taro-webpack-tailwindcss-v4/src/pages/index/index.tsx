import { Text, View } from '@tarojs/components'
import { useLoad } from '@tarojs/taro'
import './index.css'

export default function Index() {
  useLoad(() => {
    console.log('Page loaded.')
  })

  return (
    <>
      <View className="bg-[#2e2bcc] text-[100rpx] text-white">
        <Text>Hello world!</Text>
      </View>
      <View className="bg-purple-800 text-pink-200">
        11
      </View>
      <View className="h-14 bg-gradient-to-r from-cyan-500 to-blue-500"></View>
      <View className="bg-purple-800 text-pink-200 invert">
        invert
      </View>
      <View className="bg-purple-800 text-pink-200 backdrop-blur-2xl">
        backdrop-blur-2xl
      </View>
      <View className="skew-x-0 bg-purple-800 text-pink-200">
        skew-x-0
      </View>
    </>

  )
}
