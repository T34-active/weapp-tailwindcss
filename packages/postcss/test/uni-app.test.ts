import { createStyleHandler } from '@/index'
import fs from 'fs-extra'
import path from 'pathe'

describe('uni-app', () => {
  it('postcss', async () => {
    const styleHandler = createStyleHandler({
      isMainChunk: true,
    })
    const code = await fs.readFile(path.resolve(__dirname, './fixtures/css/uni-app.css'), 'utf8')
    const { css } = await styleHandler(code, {
      isMainChunk: true,
    })
    expect(css).toMatchSnapshot()
  })
})
