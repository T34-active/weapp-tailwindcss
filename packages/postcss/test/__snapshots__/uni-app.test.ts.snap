// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`uni-app > postcss 1`] = `
"/* #ifdef MP-WEIXIN */
.wx-color {
  color: aliceblue;
}

/* #endif */"
`;
